SetupDataPkg/Docs/PlatformIntegration/PlatformIntegrationSteps.md

```diff
@@ -103,57 +98,74 @@ The platform must define `MU_SCHEMA_DIR` and `MU_SCHEMA_FILE_NAME` in PlatformBu
 
-## Configuration App Code Integration
+The PlatformConfigDataLib includes, in this order, the standard structure definition header, the client header, and the
+data header. In this way the structure and enum definitions are populated before the data header attempts to use them.
 
-1. Ensure all submodules for the platform are based on the latest Project Mu version (e.g. "release/202208")
-1. The prerequisites of this feature is Policy services and Project MU based BDS as well as DFCI features. This guideline
-will omit the integration steps for these features. For more information about DFCI integration, please see [here](https://microsoft.github.io/mu/dyn/mu_plus/DfciPkg/Docs/PlatformIntegration/PlatformIntegrationOverview/).
+The OEM platform config policy creator only include the standard structure definition header so that it can operate on
+the `KNOB_DATA` structure with generic data.
 
-> Note: A list of the libraries and modules made available by this package is provided in the
-  [Software Component Overview](SoftwareComponentOverview.md).
+The Config Policy to Silicon Policy Mapper(s) include, in this order, the standard structure definition header, the
+client header, and the service header. In this way the structure and enum definitions as well as the getter definitions
+are available for the code using the getters. In our mu_tiano_platforms example, the Config Policy to Silicon Policy
+Mapper is compiled with several C files that only include the client header, as the service header is only needed once
+per module to query the config knobs out of the config policy.
 
-### Platform DSC statements
+The Silicon Policy Consumers do not need to include any of the above headers and will instead fetch their configuration
+directly from silicon policy.
 
-Add the DSC sections below.
+### Config Knob Validation Functions
 
-> Note: This is change is on top of Project MU based BDS and DFCI feature.
+The Project Mu XML based configuration provides an OEM/platform with the ability to provide config knob validation
+functions to run against config knob overrides to ensure they meet the requirements for a knob. These functions can
+be added to the `KNOB_DATA` structure's `Validator` field, say in PlatformConfigDataLib and then run in the OEM
+Config Policy Creator after it fetches overrides. If any overrides fail validation (say a value too large), the default
+config knob value can be used instead.
+
+## Configuration App Code Integration
+
+1. Ensure all submodules for the platform are based on the latest Project Mu version (check the HEAD of each repo)
+1. The prerequisites of this feature is Policy service and Project MU based BDS. This guideline will omit the
+integration steps for these features.
+
+### Platform DSC statements
+
+Add the DSC sections below in addition to the OemConfigPolicyCreator, PlatformConfigDataLib, Default Silicon Policy
+Creators, and Config Policy to Silicon Policy Mapper(s).
 
 ``` bash

```

Comment:
I don't think it makes sense, because some of them need to be overridden by the platform (the three GUID ones), so there are a couple things that could be in a dsc.inc, but not everything that is needed and not all that much.